src/App.tsx

```diff
@@ -1,10 +1,52 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { SortEnum } from './types/sort';
 
-const USER_ID = 0;
+const USER_ID = 10589;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [select, setSelect] = useState('all');
+  const [isError, setIsError] = useState(false);
+  // const [isDeleteError, setDeleteIsError] = useState(false);
```

Comment:
it's better remove comments (you will remember it till the next task : ) )
also, talking about errors, I'd recommend creating an enum with error types and setting the corresponding type in the state instead of creating a separate state for each error type

but for now, you can keep `const [isError, setIsError] = useState(false);`

FYI